packages/perseus/src/widgets/interactive-graph.tsx

```diff
@@ -1826,6 +1826,7 @@ class InteractiveGraph extends React.Component<Props, State> {
                     gridStep={gridStep}
                     snapStep={snapStep}
                     box={box}
+                    isMobile={this.props.apiOptions.isMobile}
```

Comment:
I kind of want to pass all of APIOptions and maybe have a context that lets components consume information as needed from APIOptions, but I was worried that would lead to scope creep.